docs/advanced/low-level-queries.md

```diff
@@ -0,0 +1,84 @@
+# Answering questions with custom queries
+
+The data that the subgraphs have can be used to answer more complex questions; However, you'll need a more complex query.
+The available connectors create a simplified abstraction over the data, implementing a subset of queries that can cover most of the generic use cases.
+
+## Subgraph schemas
+
+Understanding the subgraph's schema we are going to fetch data from is key so we can ask the right questions. In the [Connectors Reference](../connectors/organization.md) section you can find the schema for each subgraph associated with its particular connector.
+
+## Creating a custom query
+
+You can create custom queries to exploit the full power of the data contained in the subgraph. Following is an example of how to do it:
+
+```javascript
+import gql from 'graphql-tag'
+import { GraphQLWrapper } from '@aragon/connect-thegraph'
+
+// Construct the query
+const QUERY = `
+query {
+  votes(where:{
+    appAddress: "${VOTING_APP_ADDRESS}"
+  }) {
+    id
+    metadata
+    creator
+  }
+}
+`
+
+// Create the GraphQL wrapper using the specific subgraph URL
+const wrapper = new GraphQLWrapper(VOTING_SUBGRAPH_URL)
+
+// Fetch the data performing the custom query
+const results = await wrapper.performQuery(gql(QUERY))
```

Comment:
I don’t think `gql(QUERY)` would work? We should probably do this instead:

```jsx
const QUERY = gql`
  # …
`
```

Reply:
I like more the option you propose. Regardless the other way also works because I check on 1Hive's honey-pot and it's writing like `gql(QUERY)`

Reply:
Interesting, I imagine `gql` checks if it receives an array of strings or a string, and adapt its behavior to work in both cases :thinking: